scripts/perf/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:buster-20200514-slim
+
+RUN apt-get update \
+    && apt-get install -y \
+        git \
+        nvme-cli \
+        build-essential
+
+
+WORKDIR /root
+
+# Clone the fio repo
+RUN git clone https://github.com/axboe/fio
+RUN cd fio/ && ./configure && make
+
+# Clone the SPDK repo
+RUN git clone https://github.com/spdk/spdk.git
+RUN git -C /root/spdk submodule update --init
```

Comment:
this can be done in a single command by passing the `--recursive` flag to `git`. The other thing that helps reducing the amount of time it takes is to do a [shallow clone](https://www.perforce.com/blog/vcs/git-beyond-basics-using-shallow-clones). And this can also be done for submodules as well. So all this, plus what I mentioned above, in the form of a command:

```bash
git clone \
  --recursive \
  --depth 1 \
  --shallow-submodules \
  --branch v20.04.1 \
  https://github.com/spdk/spdk
```